lib/werewolf/engine/state.rb

```diff
@@ -0,0 +1,46 @@
+module Werewolf
+  module Engine
+    class State
+      attr_reader :current, :turn_number, :day_number, :enumerator
```

Comment:
i'm slightly struggling with the naming of `current`.  The class is `State`, and encapsulates both the day and time, but `current` only refers to the time, which is throwing me off.  I think might be more clear as `time`, `current_time`, or `time_of_day`?